sentry_sdk/integrations/aiohttp.py

```diff
@@ -115,6 +115,9 @@ async def sentry_app_handle(self, request, *args, **kwargs):
                     except asyncio.CancelledError:
                         transaction.set_status("cancelled")
                         raise
+                    except ConnectionResetError:
+                        transaction.set_status("connection_reset")
```

Comment:
the span status needs to be one in the list defined [here](https://develop.sentry.dev/sdk/event-payloads/span/#attributes)

Reply:
Thanks for reviewing my pull request!  I've updated the patch to consider `ConnectionResetError` to have "cancelled" status.